proofs/signatures/th_lira_test.plf

```diff
@@ -249,7 +249,7 @@
   (@ f2 (>=_Real
           (+_Real (term_real_var x) (term_int_to_real (term_int_var y)))
           (a_real 0/1))
-  (@ f3 (not (>=_IntReal (*_Int (a_int 1) (term_int_var y)) (a_real 0/1)))
```

Comment:
These changes align the example statements in this test with the statements that arithmetic will actually produce.
It will print the literal as an integer, and then convert it to a real higher up.